Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This is nginx 1.24.0 on bullseye.
+# https://hub.docker.com/layers/nginxinc/nginx-unprivileged/1.24.0-bullseye/images/sha256-a8ec652916ce1e7ab2ab624fe59bb8dfc16a018fd489c6fb979fe35c5dd3ec50
+FROM docker.io/nginxinc/nginx-unprivileged:1.24.0-bullseye@sha256:ac0654a834233f7cc95b3a61550a07636299ce9020b5a11a04890b77b6917dc4
+
+LABEL org.opencontainers.image.title frames
+LABEL org.opencontainers.image.source https://github.com/tkhq/frames
+
+COPY nginx.conf /etc/nginx/nginx.conf
+
+COPY recovery /usr/share/nginx/recovery
+COPY export /usr/share/nginx/export
```

Comment:
I don't think these `COPY` statements will be reproducible (i.e. the timestamps of the directories will change each time this runs)

I'm not 100% sure we need to address that now, but I think the fix would be something like:
1. COPY into a `/newroot` 
2. fix timestamps in `/newroot`
3. Do a `FROM scratch` + `COPY --from` to flatten the COPYs away

Reply:
Yeah it's probably something we'll want to address in a followup, either with this trick or by getting toolchain involved: tracking this here: https://github.com/tkhq/frames/issues/13